client/src/components/QuestionsAnswers.jsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import QuestionsSearch from './Sub_Questions/QuestionSearch.jsx';
 import QuestionList from './Sub_Questions/QuestionList.jsx';
 import AskQuestion from './Sub_Questions/AskQuestion.jsx';
 import AnswerQuestion from './Sub_Questions/AnswerQuestion.jsx';
 
 export default function QuestionsAnswers() {
+  const tempId = 40355;
+  const [questionList, setQuestionList] = useState([]);
+  const [filteredList, setFilteredList] = useState([]);
+
+  useEffect(() => {
+    axios.get(`/api/qa/questions?product_id=${tempId}&count=5&page=1`)
+      .then((result) => {
+        setQuestionList(result.data.results);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
```

Comment:
on line 21, you may want to pass the productID "tmpId" into the array as second argument of the useEffect. So when the tmpId changes, the QA component can get updated.

Reply:
nice catch! thank you, I completely got myself into working with only 1 hard coded data.